server/src/schemas/holdsSchema.ts

```diff
@@ -21,6 +22,7 @@ export const HoldsTypeDefs = gql`
     id: Int!
     type: HoldType!
     description: String
+    createdBy: Int!
```

Comment:
Why is this an int? Is it the user ID of the person who created it? If so, can we rename to something like `creatorId`?

Reply:
@MatthewGalan it does return a `User` object in the type. The corresponding input type requires an `Int` because we just need the ID of the person who created it.